webapp/src/components/modals/close_reopen_issue/index.tsx

```diff
@@ -61,12 +58,12 @@ const CloseOrReopenIssueModal = ({theme}: {theme: Theme}) => {
     const handleIssueCommentChange = (updatedComment: string) => setComment(updatedComment);
 
     const style = getStyle(theme);
-    const modalTitle = messageData.status + ' Issue';
-    const savingMessage = messageData.status === 'Close' ? 'Closing' : 'Reopening';
-    const status = messageData.status + ' Issue';
+    const modalTitle = messageData.status === 'open' ? 'Close Issue' : 'Open Issue';
+    const savingMessage = messageData.status === 'open' ? 'Closing' : 'Reopening';
+    const status = messageData.status === 'open' ? 'Close Issue' : 'Open Issue';
```

Comment:
We can create a constant for this before hand as we are performing the same action twice